package.json

```diff
@@ -27,7 +27,8 @@
     "@ngx-translate/http-loader": "^4.0.0",
     "@fortawesome/angular-fontawesome": "^0.3.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.12",
-    "@fortawesome/free-solid-svg-icons": "^5.6.3"
+    "@fortawesome/free-solid-svg-icons": "^5.6.3",
+    "rxjs": "^6.3.3"
```

Comment:
why is rxjs added there?

Reply:
Rxjs is being used @ 
https://github.com/jhipster/ng-jhipster/blob/9e10c4fae4bb3c5643dfe6f40ef91b460357b41f/src/service/event-manager.service.ts#L20

Reply:
Then we should add `rxjs` as a `devDependencies` also :)

Reply:
Oh sorry didn't see !
Can you then please add the same `~` selector ? To be consistent between the dev and peer dependencies :)

Reply:
updated to use caret in dev dependencies